api/build.gradle

```diff
@@ -322,6 +346,8 @@ ideaModule.dependsOn generateMandrillClient
 compileGeneratedJava.dependsOn generateMandrillClient
 ideaModule.dependsOn generateMoodleClient
 compileGeneratedJava.dependsOn generateMoodleClient
+ideaModule.dependsOn generateRdrClient
```

Comment:
Make sure you can run tests in IntelliJ without doing external steps in gradle, if possible.